modules/tinymce/src/plugins/table/test/ts/browser/ClipboardTest.ts

```diff
@@ -421,6 +421,182 @@ UnitTest.asynctest('browser.tinymce.plugins.table.ClipboardTest', (success, fail
     );
   });
 
+  suite.test('TestCase-TINY-6006: Table: mceTablePasteColBefore command', (editor) => {
```

Comment:
When we did TINY-6006 previously, it seems we missed adding some TinyMCE specific tests. As such I've added a number of important ones here as well.